src/jupyter_contrib_nbextensions/nbextensions/notify/notify.js

```diff
@@ -95,7 +104,15 @@ define(["require"], function (require) {
   var notify = function () {
     var elapsed_time = current_time() - start_time;
     if (enabled && !first_start && !busy_kernel && elapsed_time >= min_time) {
-      var n = new Notification(IPython.notebook.notebook_name, {body: "Kernel is now idle\n(ran for " + Math.round(elapsed_time) + " secs)"});
+      var opts = {
+        body: "Kernel is now idle\n(ran for " + Math.round(elapsed_time) + " secs)",
+        icon: "/static/base/images/favicon.ico",
```

Comment:
Neat, but to work correctly with non-default base_url (e.g. on jupyterhub), this should be 

```javascript
Jupyter.notebook.base_url + 'static/base/images/favicon.ico'
```